developersociety/commonslibrary
# Updated colour accent from green to blue

# Sources
https://trello.com/c/1nNmVZQE/27-5-can-all-the-green-borders-and-the-load-more-button-be-in-the-open-blue

# Description
This PR updates the colour accent of the site from green to the new blue hex code provided. 

# Setup instructions
n/a

# How to test
Go to http://127.0.0.1:8000/ and compare it to the live site https://thecampaignslibrary.com/. Check that the accents that were green (such as the green borders, load more button and nav bar buttons) are now showing as blue locally. 

(Note: Although the partner didn't specify every single green accent to be changed they have said they're happy for me to make the change I think is best, so I went for changing the accent colour completely to avoid a colour clash, especially as that shade of green is no longer part of their colour palette.)

For comprehensive guidelines on code review at DEV, see here: https://www.notion.so/developersociety/Code-Review-f804361f9b6b4859bdcd69d7704168e5.

# Pre-review checklist
